examples/connext_dds/asynchronous_publication/java/README.md

```diff
@@ -1,77 +1,82 @@
 # Example Code: Asynchronous Publishing
 
-## Building Java Example
+## Building the Example :wrench:
 
-Before compiling or running the example, make sure the environment variable
-`NDDSHOME` is set to the directory where your version of *RTI Connext* is
-installed.
+To build this example, first set all the environment variables as follows:
 
-Run *rtiddsgen* with the `-example` option and the target architecture of your
-choice (e.g., *i86Win32VS2010* or *i86Linux2.6gcc4.4.5*). The *RTI Connext Core
-Libraries and Utilities Getting Started Guide* describes this process in detail.
-Follow the same procedure to generate the code and build the examples. **Do not
-use the `-replace` option.** Assuming you want to generate an example for
-*i86Win32VS2010* run:
+On *UNIX* systems run:
 
 ```sh
-rtiddsgen -language Java -example i86Win32VS2010 async.idl
+<install_dir>/recource/scripts/rtisetenv_<target>.<bash|tcsh|zsh>
```

Comment:
You need to source the script to modify the environment of the current shell, so on UNIX systems you need to:

```sh
source <install_dir>/resource/scripts/rtisetenv_<target>.<bash|zsh|tcsh>
```